src/renderer/cluster-frame.tsx

```diff
@@ -101,7 +102,11 @@ export class ClusterFrame extends React.Component {
 
     catalogEntityRegistry.activeEntity = ClusterFrame.clusterId;
 
-    extensionLoader.loadOnClusterRenderer();
+    // Only load the extensions once the catalog has been populated
+    when(
+      () => Boolean(catalogEntityRegistry.activeEntity),
```

Comment:
What's the ramification if this doesn't become true, due to a bug, say? Should there be a log after some timeout?

Reply:
The ramification is that the extensions do not load. Sure I'll add a notification